README.md

```diff
@@ -145,6 +145,8 @@ option would have no effect.
 
 ## Setting Mock Locations
 
+Please set the Appium Settings from the Settings app's _Developer Options_ -> _Select mock location app_. 
```

Comment:
I assume this could also be done via adb?

Reply:
yea, recalled we did via appops. Let me add it in this readme